Backpack/Calendar/Classes/BPKCalendarCell.m

```diff
@@ -164,9 +171,16 @@ - (void)layoutSubviews {
 - (void)configureAppearance {
     [super configureAppearance];
 
+    UIColor *rangeColor = self.appearance.selectionColor;
     UIColor *selectedColor = self.preferredTitleSelectionColor ?: self.appearance.titleColors[@(FSCalendarCellStateSelected)];
     UIColor *color = self.preferredTitleDefaultColor ?: [self colorForCurrentStateInDictionary:self.appearance.titleColors];
 
+    self.selectionLayer.fillColor = rangeColor.CGColor;
```

Comment:
Does this work correctly if the user switches appearance(light vs dark) while the calendar is on screen? `CGColor` isn't appearance aware like `UIColor`

Reply:
Yeah it does 👍